spec/requests/ordering_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+describe 'Ordering' do
+  let(:proper_params) { { user_name: 'user', text: 'trigger txt', trigger_word: 'trigger' } }
+  let(:wrong_params) { { user_name: 'user', text: 'trigger anything_else', trigger_word: 'trigger' } }
+
+  let!(:place) { FactoryGirl.create(:place) }
+  let!(:order_list) { FactoryGirl.create(:order_list, place: Place.last) }
+  let!(:order) { FactoryGirl.create(:order, order_list: OrderList.last) }
+
+
+  context 'Order list' do
+    it 'should create ordering place' do
+      post 'api/orders/place', proper_params
+      expect(last_response.status).to eq(200)
+      expect(OrderList.count).to eq(1)
+      orderlist = OrderList.last
+      expect(orderlist.place).to eq(Place.last)
+    end
+
+    it 'should not create order list due to still active previous one' do
+      expect(OrderList.count).to eq(1)
+      post 'api/orders/place', proper_params
+      expect(last_response.status).to eq(200)
+      expect(OrderList.count).to eq(1)
+    end
+
+    it 'should not create order list due to wrong place name' do
+      expect(OrderList.count).to eq(1)
+      post 'api/orders/place', wrong_params
+      expect(last_response.status).to eq(200)
+      expect(OrderList.count).to eq(1)
+    end
+  end
+
+  context 'New order' do
+    it 'should create new order' do
+      post 'api/orders/new', proper_params
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.